job_definitions/update_api_tokens.yml

```diff
@@ -0,0 +1,74 @@
+{% set stages = [{'name': 'Add new tokens', 'command': 'add-new'}, {'name': 'Remove old tokens', 'command': 'remove-old'}] %}
+{% set app_groups = [['search-api'], ['api'], ['admin-frontend', 'briefs-frontend', 'brief-responses-frontend', 'buyer-frontend', 'supplier-frontend', 'user-frontend']] %}
+---
+- job:
+    name: update-api-tokens
+    display-name: "Update API tokens"
+    project-type: pipeline
+    descript1ion: Update Data and Search API tokens for a given environment.
```

Comment:
Could this be slightly more descriptive please i.e. what does 'update' mean

Reply:
Maybe we could call this 'rotate_api_tokens' instead of update?

Reply:
👍